src/test/java/testhelper/PerformanceUtils.java

```diff
@@ -0,0 +1,28 @@
+package testhelper;
+
+import com.sun.management.ThreadMXBean;
+
+import java.lang.management.ManagementFactory;
+
+public class PerformanceUtils {
+
+  private static ThreadMXBean THREAD_MX_BEAN;
+
+  static {
+    Object bean = ManagementFactory.getThreadMXBean();
```

Comment:
Curious. Why get it first as an object, run an instance check and then perform the casting ? The signature of `getThreadMXBean()` states it would return a `ThreadMXBean`.

```java
public static ThreadMXBean getThreadMXBean() {
    return ManagementFactoryHelper.getThreadMXBean();
}
```